joomla/joomla-cms
# [html5] Hathor administrator template

#### Summary of Changes

This is a redo of https://github.com/joomla/joomla-cms/pull/9842 for hathor only.

This PR does a basic conversion of **hathor** template to HTML5.
Also made some minor corrections:
- Use script/style version for template js/css
- Use the API in inline JS/CSS (except in error pages)
- Use html5.js for browser lower than IE9 in all files.
#### Testing Instructions
1. Install patch
2. Set the admin tempalte as "Hathor"
3. Go to admin and check if the the following pages are working properly:
   - normal page (test any existent URL)
   - component popup (add `&tmpl=component&print=1` to an existent page)
   - error pages (test one non existent URL)
   - login (logoff and check the login page)

Check also the code changes.
#### Observations

All pages should now start with this code:

``` html
<!DOCTYPE html>
<html lang="en-gb" dir="ltr">
<head>
[...]
    <meta charset="utf-8" />
[...]
```
